templates/components/form/form.jsx

```diff
@@ -65,7 +65,7 @@ var Form = React.createClass({
     );
 
     if (value.type === 'checkbox') {
-      input = (<span>{input}<span/></span>);
+      input = (<span className={value.className}>{input}<span/></span>);
```

Comment:
If it's undefined then it will just not add any class at all which shouldn't be a problem here since I've tested with other pages and we don't have any problem... It's also because we only use check box in sign up page too I guess...